view.go

```diff
@@ -62,13 +62,16 @@ func (v *View) UpdateWithSize(width, height int) {
 		v.Width = width
 		v.isDirty = true
 	}
-
 	v.Update()
 }
 
 // Draw draws the view
 func (v *View) Draw(screen *ebiten.Image) {
 	v.containerEmbed.Draw(screen)
+
+	if Debug {
```

Comment:
Should we change the if condition as below? I thought this way we can reduce the draw count for performance :) What do you think?
```go
	if Debug && !v.hasParent {
		debugBorders(screen, v.containerEmbed)
	}
```

Reply:
Yep, 100%, just updated this